generators/kubernetes/prompts.js

```diff
@@ -102,3 +104,64 @@ function askForIngressDomain() {
         done();
     });
 }
+
+function askForIstioSupport() {
+    if (this.composeApplicationType === 'monolith') {
+        this.istioSupportOpt = 'no';
+        return;
+    }
+    const done = this.async();
+
+    const prompts = [{
+        type: 'list',
+        name: 'istioSupportOpt',
+        message: 'Choose your preference for Istio Service Mesh',
```

Comment:
I would use the following:
- name: `istio`
- message: `Do you want to setup Istio ?`
- value: 'manualInjection`or value: `automaticInjection`